### PR TITLE
`SwiftReleasesChecker` to report new Swift releases

### PR DESCRIPTION
**Pull request by MahdiBM**

### CODE REVIEW

File: Sources/Penny/SwiftReleasesChecker.swift [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    let discordService: DiscordService
+    let logger = Logger(label: "SwiftReleasesChecker")
+
+    init(swiftReleasesService: any SwiftReleasesService, discordService: DiscordService) {
+        self.swiftReleasesService = swiftReleasesService
+        self.discordService = discordService
+    }
+
+    nonisolated func run() {
+        Task { [self] in
```
Comment (0xTim) [inline Sources/Penny/SwiftReleasesChecker.swift:27]: Why are we doing this with unstructured concurrency?

Comment (MahdiBM) [reply Sources/Penny/SwiftReleasesChecker.swift:27]: I was thinking about this. I should move things to ServiceLifecycle, but that'll have to be for another PR.

Comment (MahdiBM) [reply Sources/Penny/SwiftReleasesChecker.swift:27]: Everything matches ServiceLifecycle too, coincidentally. Even the function name, other than the fact that the function is a "blocking" function which is what ServiceLifecycle expects.

Comment (MahdiBM) [reply Sources/Penny/SwiftReleasesChecker.swift:27]: This pattern is what we use everywhere in Penny so can't just fix it in one place, Will need more changes.

Comment (MahdiBM) [reply Sources/Penny/SwiftReleasesChecker.swift:27]: https://github.com/vapor/penny-bot/pull/255

**Review by 0xTim — APPROVED** [review]

**Pull request merged**